src/binder/inc/clrprivbinderassemblyloadcontext.h

```diff
@@ -46,15 +71,37 @@ class CLRPrivBinderAssemblyLoadContext : public IUnknownCommon<ICLRPrivBinder>
             /* [out] */ HRESULT *pResult,
             /* [out] */ ICLRPrivAssembly **ppAssembly);
 
+#ifdef FEATURE_COLLECTIBLE_ALC
+
+    //=========================================================================
+    // IAssemblyLoadContext functions
+    //-------------------------------------------------------------------------
+    STDMETHOD(GetIsCollectible)(
+            /* [retval][out] */ BOOL *pIsCollectible);
+
+    STDMETHOD(ReferenceLoaderAllocator)(
+            /* [in] */ LoaderAllocator *pLoaderAllocator);
+
+#endif // FEATURE_COLLECTIBLE_ALC
+
 public:
     //=========================================================================
     // Class functions
     //-------------------------------------------------------------------------
 
-    static HRESULT SetupContext(DWORD      dwAppDomainId, CLRPrivBinderCoreCLR *pTPABinder, 
-                                UINT_PTR ptrAssemblyLoadContext, CLRPrivBinderAssemblyLoadContext **ppBindContext);
-                    
+    static HRESULT SetupContext(AppDomain *pAppDomain, CLRPrivBinderCoreCLR *pTPABinder, 
```

Comment:
As mentioned in the other comment, it is not clear why AppDomain* is being passed as opposed to the ID.